### PR TITLE
add variables to choose what this role do or not

### PR DESCRIPTION
**Pull request by nicolas-san**
I have a server already full configured, but this role is perfect to install suiteCRM, so I decide to use it and add variable to allow me to disable some part of it.
Like the apt part, or the composer install
I move the repair.php tool to the templates directory to change the path for the includes_once
Tested on my dev environment

### CODE REVIEW

**Review by stancel — APPROVED** [review]
I am going to merge in all of these changes, but then overwrite one in a commit right after merging. I am keeping the vars/main.yml file. The reason is because all of the roles I write and maintain have this file available so that I can easily copy and paste only the required variables into a similar file I use when making playbooks. That way all variables for all roles in that playbook are managed in one variable file that is referenced as an argument when executing the playbook, giving those variables the highest level of precedence. Thanks for reaching out and contributing back! Much appreciated!

**Comment by nicolas-san** [comment]
Correct me if I'm wrong, but when we use the role from the ansible galaxy, as a dependence, we do not edit the vars/main.yml
My vars are defined in the ./site.yml file and have not the precedence according to https://gist.github.com/ekreutz/301c3d38a50abbaad38e638d8361a89e
Where can I add my vars to have to best way to do it and have the precedence over the vars/main.yml ?
In the readme you say copy the vars/main.yml but the first time I use it, I just add the role from ansible galaxy to my requirements and tried to set my vars

Thanks again for the initial work, and the merging :)

**Pull request merged**
